### ISSUE TITLE
OS X 10.11, El Capitan installation not possible with SIP enabled.

### ISSUE THREAD

**Issue by ghost** [jolie/release_tools#2]
It is not possible to install Jolie as described on the webpage in OS X 10.11, El Capitan. This is due to Apple having made 'usr/lib/' protected with System Integrity Protection (SIP) and users are not able to write to it even using sudo. Jolie can be installed as before if SIP is disabled.

Disableing SIP:

Start Mac in recovery mode - when starting up hold: cmd + r

In recovery mode open terminal (under utilities)

In terminal run command: csrutil disable

Then restart computer 

Then Jolie can be installed as before

**Comment by fmontesi** [comment]
@thesave Argh.. :-(

@IlCakeBomber @thesave What's the correct way of installing system-level stuff on OS X now? I guess the brew people had to address this problem? (and all others trying to install stuff)

**Comment by thesave** [comment]
Thanks for reporting the issue @IlCakeBomber.
A few days ago I committed an update for the download page of the Jolie website [1] with a FAQ on OS X 10.11.
The gist of it is to use the "local" version of /usr/bin and /usr/lib.
I tested that solution and it seems to work properly.
I did not try the solution proposed by @IlCakeBomber.
Maybe we can add it as an alternative in the related FAQ?

[1] https://github.com/jolie/website/commit/a98775bba5b6c4699935702109b6d1161454ce47
